frontend/src/hooks/CreateRoom/useSectionValidation.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+const useSectionValidation = (isCalendarReady: boolean, isBasicReady: boolean) => {
+  const showValidation = useRef(false);
+  const [shouldShake, setShouldShake] = useState(false);
+
+  const validateSection = useCallback(() => {
+    showValidation.current = true;
+
+    if (!isCalendarReady) {
+      setShouldShake(true);
+      return false;
+    }
+    if (!isBasicReady) {
+      setShouldShake(true);
+      return false;
+    }
+    return true;
+  }, [isCalendarReady, isBasicReady]);
+
+  useEffect(() => {
+    if (shouldShake) {
+      const timer = setTimeout(() => {
+        setShouldShake(false);
+      }, 460);
+
+      return () => clearTimeout(timer);
+    }
+  }, [shouldShake]);
+
+  return {
+    showValidation,
```

Comment:
```suggestion
showValidation:showValidation.current
```

이렇게 해서 외부에서는 boolean값만 쓰게 하는건 어떨까요?